src/com/musala/xml/parser/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.musala.xml.parser;
+
+class Constants {
+    static String filepath = "ThePath.xml";
```

Comment:
By convention. constants must be named using the pattern UPPERCASE_LETTERS_USED_SEPARATED_BY_UNDERSCORES. The constant name should be something like MAIN_XML_FILE_PATH, XML_FILE_PATH, etc.